charts/dependency-track/templates/api-server/statefulset.yaml

```diff
@@ -95,6 +95,13 @@ spec:
       {{- if .Values.apiServer.extraContainers }}
       {{- toYaml .Values.apiServer.extraContainers | nindent 6 }}
       {{- end }}
+      securityContext:
+        fsGroup: 1000
```

Comment:
Would this need to be configurable in order to remain compatible with OpenShift's random GIDs?

Reply:
Just realized making it configurable might not even be beneficial, as the random GID is only assigned at runtime. Any common pattern on how to deal with this?

Reply:
Hi @nscuro :)

I'm not familiar with OpenShift... . The issue was that /data had the owner set to root:root, which prevented the API from writing to it as the user dtrack (uid/gid 1000). 
What do you think about adding an initContainer that runs chown -R 1000:1000 /data instead of setting securityContext? From what I can see, this approach is used by, for example, Grafana. But as I understand, this won't help with OpenShift, right?

Reply:
Also please remember to sign-off your commits. The currently failing DCO check will give instructions on how to do it when clicking the *Details* link next to it.